Private/Get-CultureInfo.ps1

```diff
@@ -7,13 +7,13 @@ function Get-CultureInfo {
     )
 
     $LanguageName = $LanguageName.Trim()
-    [cultureinfo]$CultureInfo = [cultureinfo]::GetCultures([System.Globalization.CultureTypes]::AllCultures) |`
+    [cultureinfo]$CultureInfo = [cultureinfo]::GetCultures([System.Globalization.CultureTypes]::AllCultures) |
         Where-Object {
-        $_.Name -eq $LanguageName `
-            -or $_.EnglishName -eq $LanguageName `
-            -or $_.DisplayName -eq $LanguageName `
-            -or $_.NativeName -eq $LanguageName `
-            -or $_.TwoLetterISOLanguageName -eq $LanguageName `
+        $_.Name -eq $LanguageName
+            -or $_.EnglishName -eq $LanguageName
+            -or $_.DisplayName -eq $LanguageName
+            -or $_.NativeName -eq $LanguageName
+            -or $_.TwoLetterISOLanguageName -eq $LanguageName

```

Comment:
This code looks incorrect; the position of the or operator needs to be changed from the beginning of the line to the end of the line. Like this.

```
$_.Name -eq $LanguageName -or
  $_.EnglishName -eq $LanguageName -or
  $_.DisplayName -eq $LanguageName -or
  $_.NativeName -eq $LanguageName -or
  $_.TwoLetterISOLanguageName -eq $LanguageName
```

But this code is bit difficult to read, so it may be better to keep the original which uses backticks.